topics/icrc_32_sign_challenge.md

```diff
@@ -0,0 +1,126 @@
+# ICRC-32: Sign Challenge
+
+[![Status Badge](https://img.shields.io/badge/STATUS-DRAFT-ffcc00.svg)](https://github.com/orgs/dfinity/projects/31)
+[![Extention Badge](https://img.shields.io/badge/Extends-ICRC--25-ffcc222.svg)](./icrc_25_signer_interaction_standard.md)
+
+<!-- TOC -->
+
+- [ICRC-32: Sign Challenge](#icrc-32-sign-challenge)
+  - [Summary](#summary)
+  - [Method](#method)
+  - [Request](#request)
+    - [Example RPC Request ](#example-rpc-request)
+  - [Response](#response)
+    - [Example RPC Response ](#example-rpc-response)
+  - [Message Processing](#message-processing)
+  - [Errors](#errors)
+
+## Summary
+
+The purpose of the `icrc32_sign_challenge` method is for the relying party to receive a cryptographic proof of ownership for the users identities.
+
+## Method
+
+**Name and Scope:** `icrc32_sign_challenge`
+
+**Prerequisite:** Active session with granted permission scope `icrc32_sign_challenge` or `*`.
+
+## Request
+
+**`version` (`text`):** The version of the standard used. If the signer does not support the version of the request, it must send the `"VERSION_NOT_SUPPORTED"` error in response.
+
+**`challenge` (`blob`):** A challenge used for the signer to sign in order to prove its access to the identity. The challenge should be an array of 32 cryptographically random bytes generated from a secure random source by the sender of the request.
+
+### Example RPC Request
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "icrc32_sign_challenge",
+  "params": {
+    "version": "1",
+    "principal": "rwlgt-iiaaa-aaaaa-aaaaa-cai",
+    "challenge": "UjwgsORvEzp98TmB1cAIseNOoD9+GLyN/1DzJ5+jxZM="
+  }
+}
+```
+
+## Response
+
+`version` (`text`): The version of the standard used. It must match the `version` from the request.
+
+`identities`: A list of identities the user has selected to share with the relying party.
```

Comment:
I think we should rename this property to `signedChallenge` (or similar) and make it just an object (rather than a list) to better fit the new method.